news/10128.doc.rst

```diff
@@ -0,0 +1 @@
+Improve the text of the deprecation warning regarding switching to in-place builds.
```

Comment:
```suggestion
Improve deprecation warning regarding the copying of source trees when installing from a local directory.
```

Reply:
Let's also change this to a `.removal.`, since this is related to a deprecation warning (those entries show up under a "Deprecations and Removals" heading).

Reply:
Is `.removal.` a newsfile type? So it would replace `.doc.`?

Reply:
Yup.

Reply:
The types are listed here: https://github.com/pypa/pip/blob/main/pyproject.toml#L13